apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -288,6 +292,12 @@
     <div class="ui divider hidden"></div>
 <% } %>
 
+<% if (isAdminAdvisoryBannerEnabledInTenant) { %>
+    <div class="ui warning message" data-componentid="login-page-admin-session-advisory-banner">
+        <%=adminAdvisoryBannerContentOfTenant%>
```

Comment:
Shouldn't this encode (if `adminAdvisoryBannerContentOfTenant` is an user input)
```suggestion
        <%=Encode.forHtml(adminAdvisoryBannerContentOfTenant)%>
```